bedrock/releasenotes/migrations/0001_initial.py

```diff
@@ -28,7 +28,7 @@ class Migration(migrations.Migration):
                 ("system_requirements", bedrock.releasenotes.models.MarkdownField(blank=True)),
                 ("created", models.DateTimeField()),
                 ("modified", models.DateTimeField()),
-                ("notes", bedrock.releasenotes.models.NotesField(blank=True)),
+                ("notes", models.JSONField(blank=True)),
```

Comment:
Oh yeah, I missed that this isn't the same `JSONField` in the `models.py`.

Reply:
Oh yeah. Me too. It's `from django_extensions.db.fields.json import JSONField`

Reply:
Yeah sorted and rebased